website/content/docs/concepts/client-count/index.mdx

```diff
@@ -253,3 +253,58 @@ For the UI to be able to use the client count feature, it needs `read` permissio
 
 For the UI to be able to modify the configuration settings, it additionally needs `update` permission to
 `sys/internal/counters/config`.
+
+## New Clients For Current Month
+
+As of Vault 1.12, the billing period for the activity log API can be specified to include the current month 
```

Comment:
Notes from Vishal sync: we should add a disclaimer/small paragraph explaining the reason for the approximation (ie why it's necessary) without getting too technical, but explaining that it's a purely technical reason, a general computing limitation.

Reply:
Make sure to stress that this is done primarily to improve API performance and make the UI usable. Mention the problem name (count-distinct for multisets) and explain why this maps to that.